### DISCUSSION TITLE
See dev domains dvl.to (from devilbox) as private domain

### DISCUSSION THREAD

**Discussion by masiorama** [craftcms/cms#15011] [Ideas] [answered]
Hello, I would like to be able to work with devilbox (my docker env tool) which with version 3 switched to `dvl.to` as default dev domain, therefore not to be bugged in admin panel by the red highlighted box.
Is it possible to grant this domain too?
Thanks in advance!

References:
- https://devilbox.discourse.group/t/devilbox-v3-0-0-beta-0-3-released-dvl-to-domain/725
- https://craftcms.com/knowledge-base/how-to-trial-craft-cms-and-plugin-editions

**Comment by brandonkelly** [comment, marked as answer]
Done!

**Reply by masiorama** [reply]
Thanks a lot, is this something I can test right now? Has to be released?

**Reply by brandonkelly** [reply]
Yep, should already work. It’s a change on the API, not Craft itself.

**Reply by masiorama** [reply]
Indeed it does work! Thanks a lot!
